tests/embind/embind.test.js

```diff
@@ -638,11 +638,13 @@ module({
             assert.equal(2, cm.const_ref_adder(true, true));
         });
 
-        assert.throws(TypeError, function() { cm.const_ref_adder(Symbol('0'), 1); });
-        assert.throws(TypeError, function() { cm.const_ref_adder(BigInt(0), 1); });
+        test("passing Symbol or BigInt as floats always throws", function() {
+            assert.throws(TypeError, function() { cm.const_ref_adder(Symbol('0'), 1); });
+            assert.throws(TypeError, function() { cm.const_ref_adder(BigInt(0), 1); });
+        });
 
-        if (cm['ASSERTIONS']) {
-            test("can pass only number and boolean with assertions assertions", function() {
+        if (cm.ASSERTIONS) {
```

Comment:
Why is this changed? (it breaks the consistency of how this is used here; and, in general for closure it is right to use the quotes)

Reply:
I went back and forth, this should not have been changed for consistency as you state.

Reply:
IIUC this code is running outside of the emscripten-generated code and outside of the scope of closure so (I think).. the quotes are maybe not needed?   But yes we should be consistent.. maybe a separate PR to remove all the quotes like this?

Reply:
If we want the test to run under closure, then it should be quoted. And I think that's a good idea, to test embind in that mode. So I'd go with quotes.

Reply:
I must be misunderstanding.  I thought closure only ran on code that was compiled with the module (i.e. --js-library code or EM_ASM code).   This code IIUC is completely external an separate and added/injected *after* compilation, so there is no way it can be effected by closure.. or am I mistaken?

Reply:
Oh, I wasn't aware of how this is used. I assumed if it's in `src` then it is compiled with the rest of the code. If it's an exception like `worker.js` then we don't run closure on it, correct.

Still, it can't hurt to have the quotes, if this is just test and not production code.

Reply:
My understanding that this is test code that hosts the emscripten module...  i.e. it lives on the outside and calls into the compiled module.  So there is no closure involved.